samples/Common.c

```diff
@@ -214,6 +215,10 @@ STATUS handleOffer(PSampleConfiguration pSampleConfiguration, PSampleStreamingSe
 
     CHK_STATUS(deserializeSessionDescriptionInit(pSignalingMessage->payload, pSignalingMessage->payloadLen, &offerSessionDescriptionInit));
     CHK_STATUS(setRemoteDescription(pSampleStreamingSession->pPeerConnection, &offerSessionDescriptionInit));
+    canTrickle = canTrickleIceCandidates(pSampleStreamingSession->pPeerConnection);
+    /* cannot be null after setRemoteDescription */
+    CHECK(!NULLABLE_CHECK_EMPTY(canTrickle));
```

Comment:
Are we trying assert here or a negative flow check? Based on the comment it's an assert so CHECK is OK